### PR TITLE
Organizing and Structuring project

### PR DESCRIPTION
**Pull request by jbeomchoi**
Since code burden was getting larger and larger, decided to split logics/views into several parts

Consider it as splitting `App.tsx` into several components, contexts, hooks

Simply, the project was splitted into 4 parts

1. Unity-related (MYTYSDKContext, MYTYSDKView)
2. Selecting cameraInput (VideoDeviceContext, VideoDeviceSelect)
3. Mediapipe (useHolistic)
4. Capture, and record (useCaptureRecord)

Since @lebrian9889 and I are both not that familiar with React, I'm open to any kind of feedback. I'm not really certain this is a good structure, though 😃 

@Hyunyoun @February24-Lee @apsntian 
I didn't add you as reviewer, but if you have time, as a senior of React project, I'd appreciate your feedbacks

### CODE REVIEW

File: examples/react_2d_mp/react/src/context/MYTYSDKContext.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            assetVersionId: assetId,
+            tokenId: tokenId
+        }))
+    }
+
+    const setARMode = (flag: string) => unityContext.sendMessage(MESSAGE_HANDLER, "SetARMode", flag)
+
+    const processCapturedResult = (data: string) => unityContext.sendMessage(MESSAGE_HANDLER, "ProcessCapturedResult", data)
+
+    const takeScreenshot = unityContext.takeScreenshot
```
Comment (apsntian) [inline examples/react_2d_mp/react/src/context/MYTYSDKContext.tsx:40]: Oh.. I didn't know this feature in react-unity-webgl. It seems cool!

File: examples/react_2d_mp/react/src/components/MYTYSDKView.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  useEffect(() => {
+    if(cameraRef.current) {
+      setCamera(cameraRef.current);
+    }
+  }, [cameraRef, setCamera])
+
+  return <div style={{ position: 'relative', width: '720px', height: '480px' }}>
+        <Unity
+          style={{
```
Comment (apsntian) [inline examples/react_2d_mp/react/src/components/MYTYSDKView.tsx:21]: Add devicePixelRatio=2 to props of Unity. The default value is 1 and it causes poor quality of the rendered result.

**Review by apsntian — COMMENTED** [review]
Sorry for late response, jay. I reviewed all code. Your code is concise and easy to understand. 

If the MYTYSDKContext become larger, please consider my abstraction for unity webgl component.

File: examples/react_2d_mp/react/src/context/MYTYSDKContext.tsx (hunk 1 above)
Comment (jbeomchoi) [reply examples/react_2d_mp/react/src/context/MYTYSDKContext.tsx:40]: Me, neither. 
It was on their API tab (https://react-unity-webgl.dev/docs/api/take-screenshot)
Actually I found this after I get to know configuration below should be applied to canvas to capture WebGL

  webGLContextAttributes: {
    preserveDrawingBuffer: true,
  },
  
 Haha... anyway it's great that they provide this

File: examples/react_2d_mp/react/src/components/MYTYSDKView.tsx (hunk 2 above)
Comment (jbeomchoi) [reply examples/react_2d_mp/react/src/components/MYTYSDKView.tsx:21]: Thank you! I'll apply it right away

**Pull request merged**